shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostToSlackHelper.scala

```diff
@@ -27,7 +27,7 @@ trait PostToSlackHelper {
   protected val slackState: SlackState
 
   protected val username: String = System.getProperty("user.name", "unknown")
-  protected val blacklistedUsernames: Set[String] = Set.empty
+  protected val excludedUsernames: Set[String] = Set.empty
```

Comment:
I think we'll need to release this as a major version bump.  This is technically a backward-incompatible change